test/functional/apps/bundles/index.js

```diff
@@ -18,15 +18,6 @@ export default function ({ getService }) {
 
     let buildNum;
     before(async () => {
-      // Wait for status to become green
-      let status;
-      const start = Date.now();
-      do {
-        const resp = await supertest.get('/api/status');
-        status = resp.status;
-        // Stop polling once status stabilizes OR once 40s has passed
-      } while (status !== 200 && Date.now() - start < 40_000);
-
```

Comment:
Are we totally sure this will not introduce flakiness?

Reply:
I'm fairly confident. With these logs in place, I don't see Kibana toggling from `available` to `unavailable`. It seems to stay stable, so we should be fine to remove this check (it was just added yesterday, so this test has been stable without this for some time).